### ISSUE TITLE
Could not find blocks and wallet file in data-dir

### ISSUE THREAD

**Issue by noprom** [EOSIO/eos#2495]
I was running a local dawn3.0 testnet using this docker image: `eosio/eos:dawn3x`, everything works fine  and the node produces nodes normally:
```
eosio generated block 72e881b2... #164684 @ 2018-04-19T02:04:05.500 with 0 trxs, lib: 164683
eosio generated block 478b51ef... #164685 @ 2018-04-19T02:04:06.000 with 0 trxs, lib: 164684
eosio generated block 662435b7... #164686 @ 2018-04-19T02:04:06.500 with 0 trxs, lib: 164685
eosio generated block 86b60c23... #164687 @ 2018-04-19T02:04:07.000 with 0 trxs, lib: 164686
eosio generated block 18170d72... #164688 @ 2018-04-19T02:04:07.500 with 0 trxs, lib: 164687
eosio generated block 1c5aa306... #164689 @ 2018-04-19T02:04:08.000 with 0 trxs, lib: 164688
eosio generated block 2d1abd93... #164690 @ 2018-04-19T02:04:08.500 with 0 trxs, lib: 164689
eosio generated block 7b5db99b... #164691 @ 2018-04-19T02:04:09.000 with 0 trxs, lib: 164690
eosio generated block e4911874... #164692 @ 2018-04-19T02:04:09.500 with 0 trxs, lib: 164691
eosio generated block 9d894796... #164693 @ 2018-04-19T02:04:10.000 with 0 trxs, lib: 164692
eosio generated block ec103757... #164694 @ 2018-04-19T02:04:10.500 with 0 trxs, lib: 164693
eosio generated block 35cc99bd... #164695 @ 2018-04-19T02:04:11.000 with 0 trxs, lib: 164694
eosio generated block 1e4f8ee5... #164696 @ 2018-04-19T02:04:11.500 with 0 trxs, lib: 164695
eosio generated block 7b4f463e... #164697 @ 2018-04-19T02:04:12.000 with 0 trxs, lib: 164696
eosio generated block d2e31dcf... #164698 @ 2018-04-19T02:04:12.500 with 0 trxs, lib: 164697
eosio generated block fd76be60... #164699 @ 2018-04-19T02:04:13.000 with 0 trxs, lib: 164698
eosio generated block 3791bf3a... #164700 @ 2018-04-19T02:04:13.500 with 0 trxs, lib: 164699
eosio generated block c5a861e2... #164701 @ 2018-04-19T02:04:14.000 with 0 trxs, lib: 164700
eosio generated block 40864f09... #164702 @ 2018-04-19T02:04:14.500 with 0 trxs, lib: 164701
eosio generated block 123e208a... #164703 @ 2018-04-19T02:04:15.000 with 0 trxs, lib: 164702
eosio generated block 1eb4f6b0... #164704 @ 2018-04-19T02:04:15.500 with 0 trxs, lib: 164703
eosio generated block e5dd093b... #164705 @ 2018-04-19T02:04:16.000 with 0 trxs, lib: 164704
eosio generated block 2bcdd42e... #164706 @ 2018-04-19T02:04:16.500 with 0 trxs, lib: 164705
eosio generated block 819be724... #164707 @ 2018-04-19T02:04:17.000 with 0 trxs, lib: 164706
eosio generated block 4dbc045c... #164708 @ 2018-04-19T02:04:17.500 with 0 trxs, lib: 164707
eosio generated block 591500b8... #164709 @ 2018-04-19T02:04:18.000 with 0 trxs, lib: 164708
eosio generated block a8175a8f... #164710 @ 2018-04-19T02:04:18.500 with 0 trxs, lib: 164709
eosio generated block 93ae134d... #164711 @ 2018-04-19T02:04:19.000 with 0 trxs, lib: 164710
eosio generated block 7c410966... #164712 @ 2018-04-19T02:04:19.500 with 0 trxs, lib: 164711
eosio generated block 7b9f4c48... #164713 @ 2018-04-19T02:04:20.000 with 0 trxs, lib: 164712
eosio generated block 6e018ccc... #164714 @ 2018-04-19T02:04:20.500 with 0 trxs, lib: 164713
eosio generated block 715d3078... #164715 @ 2018-04-19T02:04:21.000 with 0 trxs, lib: 164714
eosio generated block 888aa1da... #164716 @ 2018-04-19T02:04:21.500 with 0 trxs, lib: 164715
```
But when I was testing the wallet component, no error throws.
The strange thing is the I can not find wallet file in my `data-dir` and blocks as well:
```
root@nodeosd:/# cd /opt/eosio/bin/
root@nodeosd:/opt/eosio/bin# ls
cleos  data-dir  eosio-abigen  eosio-launcher  eosio-s2wasm  eosiocpp  keosd  nodeos  nodeosd.sh
root@nodeosd:/opt/eosio/bin# ls data-dir/
config.ini  contracts  genesis.json
```

There is no `default.wallet` and `blocks` folder under `data-dir`.
What's wrong with this?

**Comment by noprom** [comment]
This has been solved for [#2502](https://github.com/EOSIO/eos/pull/2502)

**Comment by noprom** [comment]
@yanggengyuan  I am closing this now.
